attributes/uwsgi.rb

```diff
@@ -25,5 +25,4 @@
 default['graphite']['uwsgi']['carbon'] = '127.0.0.1:2003'
 default['graphite']['uwsgi']['listen_http'] = false
 default['graphite']['uwsgi']['port'] = 8080
-default['graphite']['uwsgi']['service_type'] = 'runit'
```

Comment:
this is just a cleanup as i was thinking about the systemd portions of the cookbook, its not used anywhere. should i remove the change?